_overviews/toolkit/web-server-dynamic.md

```diff
@@ -65,17 +71,17 @@ In the terminal, the following command will start the server:
 {% endtab %}
 {% endtabs %}
 
-Access the endpoint at [http://localhost:8080/time](http://localhost:8080/time). You should see a result similar to the
-one below.
+Access [the endpoint](http://localhost:8080/time). You should see a result similar to the one below.
```

Comment:
IMO the previous text, with the link appearing in the text, was better here. We should actively show the user that we're sending them to a `localhost` URL.